src/main/java/com/stripe/model/LegalEntity.java

```diff
@@ -37,12 +52,14 @@
   @Getter
   @Setter
   @EqualsAndHashCode(callSuper = false)
-  public static class Verification extends StripeObject {
-    String details;
-    String detailsCode;
-    String document;
-    String documentBack;
-    String status;
+  public static class JapanAddress extends StripeObject {
```

Comment:
why not extend `Address` and just defined `town` as extra?

Reply:
Internally and in the OpenAPI spec, they're defined as two different schemas.

Also, at the moment there is no concrete resource class that derives from another concrete class, and I'd rather keep that invariant in place. If we had `JapanAddress` extends `Address`, and ever needed to add a field to `Address`, chances are whoever makes the change will not notice that the field would also be added to `JapanAddress`.

Reply:
Gotcha, I'm fine with that in that case.